tests/ae-inventory-subjects-itest/src/test/java/org/metaeffekt/core/itest/common/download/UrlPreparer.java

```diff
@@ -1,17 +1,24 @@
 package org.metaeffekt.core.itest.common.download;
 
+import com.google.common.io.Files;
```

Comment:
What is the benefit over using google Files here, need to understand benefit and dependency